tensorflow/tools/ci_build/ci_test_on_install.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
```

Comment:
I still don't see why this file should be here. Why not simply delete this file and run...
tensorflow/tools/ci_build/ci_build.sh GPU /tensorflow/tensorflow/tools/ci_build/builds/test_on_install.sh GPU